### DISCUSSION TITLE
conda-forge

### DISCUSSION THREAD

**Discussion by diogomart** [libocca/occa#757] [General]
Hello,

Is there interest in distributing occa through conda-forge? Not necessarily for python bindings, but for C/C++. Thinking boost-cpp might be a similar example.

Thanks

**Comment by kris-rowe** [comment]
That would be awesome. Would you be willing to help with this effort?

**Comment by diogomart** [comment]
That's great to hear. We will possibly be able to help. I'll post back here if/when I start looking into it.

**Reply by kris-rowe** [reply]
@thilinarmtb Has created a Conda package for OCCA and will give a short presentation at the TAF meeting this Wednesday.

**Reply by diogomart** [reply]
Thank you for letting me know

**Comment by atillack** [comment]
@thilinarmtb I've seen the Occa PR for Conda-Forge and was wondering if there's anything we can do to help move it along?
